aveas_openlabel/metadata.py

```diff
@@ -89,6 +89,12 @@ class Metadata(BaseMetadata):
     - Z: 'Z' if the time zone is UTC, '±[hh]:[mm]', '±[hh][mm]', or '±[hh]' otherwise, ex. '+0100'
     """
 
+    opendrive: str = field(default_factory=lambda: no_default(field="Metadata.opendrive"), metadata=required)
+    """
+    The name of the OpenDRIVE file which contains specifications that fields in this OpenLABEL refer to.
+    The OpenDRIVE file has to use the projection string indicated under "projection_string".
```

Comment:
```suggestion
    The OpenDRIVE file has to use the projection string indicated under "Metadata.projection_string".
```

The field is directly underneath, but I would reference the field with the class to match the general documentation pattern in this repo which tries to make references as precise as possible.